.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+### Description of change
+
+_Please provide description of change here_.
+
+### Pull Request check-list
+
+_Please make sure to review and check all of these items_:
+
+- [ ] Does `make test` pass after this change?
```

Comment:
"Does `make test` (Linux) or `vcbuild test nosign` pass after this change?"

Or something mentioning windows platform testing